src/datasources/targeted-messaging/targeted-messaging.datasource.ts

```diff
@@ -101,6 +101,23 @@ export class TargetedMessagingDatasource
     return this.outreachDbMapper.map(dbOutreach);
   }
 
+  async getOutreach(outreachId: number): Promise<Outreach> {
+    const [dbOutreach] = await this.sql<
+      DbOutreach[]
+    >`SELECT * FROM outreaches WHERE id = ${outreachId}`.catch((err) => {
```

Comment:
Instead of using `*`, we should explicitly specify the columns we want to fetch. Using `*` is unsafe because future changes to the table, such as adding a blob field that we don’t need to fetch here, could lead to performance issues. More about the design pattern:
https://stackoverflow.com/questions/3639861/why-is-select-considered-harmful

Reply:
https://github.com/safe-global/safe-client-gateway/commit/1b72d7acb028af5275397cac1a6b18e63e85e435#diff-7b46005236ccadec6f3c236cb143960dcc7f8c53afb914e828c907dd112384dbL107